### DISCUSSION TITLE
Enabling/Disabling Devices & Entities via a service would be a great addition to Home Assistant

### DISCUSSION THREAD

**Discussion by EpicLPer** [frenck/spook#5] [Announcements]
Starting a discussion for this cause I think that Enabling/Disabling devices or entities via a service is actually a good addition to Home Assistant.

As example, disabling a single entity that is acting up in an Automation, as example for motion sensors, to quickly debug things without having to manually enable/disable it via the UI would be a great thing! Or being able to disable entities via Zigbee Buttons and not having to split up an automation into multiple IDs or automations themselves. That would indeed be something useful!

Anyways, rambling off :)

**Comment by frenck** [comment]
🤷 Contributions to add such services have been declined in the past, hence Spook exists.
He doesn't care. 👻 

Hence: https://github.com/frenck/spook#service-disable-an-entity

../Frenck
